.github/actions/fetch-nightly-cli/action.yaml

```diff
@@ -32,6 +32,12 @@ runs:
             --json databaseId \
             --jq '.[0].databaseId'
         )
+
+        if [ -z "$run_id" ]; then
+          echo "::error::No successful workflow runs found for branch '$BRANCH'."
```

Comment:
Using the `::error::` prefix lets GitHub handle it as a special string, and shows up as a red error in the logs.

Docs: https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-commands#setting-an-error-message

Reply:
Sounds good! Since you had the urge to ocmment this here, I would say let's also add this comment to the code here, as it is quite niche fact.